demos/build_demos.sh

```diff
@@ -25,6 +25,20 @@ error() {
 }
 trap 'error ${LINENO}' ERR
 
+extra_cmake_opts=()
+
+for opt in "$@"; do
+    case "$opt" in
+    -DENABLE_PYTHON=*)
```

Comment:
`-D ENABLE_PYTHON=*` is also a valid cmake key. But I don't see easy way to handle it, so it is probably going to be kept like this.

Reply:
Yeah, I don't think we need to support the full CMake argument syntax here. It is our own script and our own option (which just happens to be a valid CMake option too :-)).